src/framework/preferences/js/PrefsEditor.js

```diff
@@ -391,13 +391,17 @@ var fluid_2_0 = fluid_2_0 || {};
             fluid.model.diff(userSelections.preferences, fluid.get(that.initialModel, ["preferences"]), stats);
 
             fluid.each(stats.changeMap, function (state, path) {
-                fluid.set(changedSelections, ["preferences", path], userSelections.preferences[path]);
+                fluid.set(changedPrefs, path, userSelections.preferences[path]);
```

Comment:
Which comment do you refer to for being at the wrong spot in the code.

To answer your question in the previous comment of "have an option to save the rest of the model or portion there of", I would say, for the data that integrators are not intended to be saved, don't relay to the prefsEditor model. The prefsEditor model is empty on its own but being populated from the initialModel whose data are meant to be tracked.